tests/testCoadds.py

```diff
@@ -175,6 +203,14 @@ def comparePsfs(self, a, b):
             for aParam, bParam in zip(aFuncParams, bFuncParams):
                 self.assertEqual(aParam, bParam)
 
+    @unittest.skip("Remove test until DM-5174 is complete")
+    def testMasksRemoved(self):
```

Comment:
I know this is waiting on another ticket, but doesn't this need more code to actually work?  Does it make sense to include it in this ticket if the later ticket will need to change it?